CMakeLists.txt

```diff
@@ -52,6 +52,7 @@ find_package(Elfutils REQUIRED)
 find_package(ECM 1.0.0 NO_MODULE REQUIRED)
 find_package(Threads REQUIRED)
 find_package(Zstd)
+find_package(objdump)
```

Comment:
the Findobjdump.cmake is missing from this patch

also, you need to add the RUNTIME dependency somewhere too, no? here's an example from a different project:

```

find_package(heaptrack QUIET)
set_package_properties(heaptrack PROPERTIES
    DESCRIPTION "A heap memory profiler for Linux"
    URL "https://phabricator.kde.org/dashboard/view/28"
    PURPOSE "Recommended: required by the non-essential heaptrack plugin"
    TYPE RUNTIME
)
```

something like that for objdump will be needed here too